substrate/client/rpc-spec-v2/src/chain_head/chain_head_follow.rs

```diff
@@ -50,6 +51,11 @@ use std::{
 /// The maximum number of finalized blocks provided by the
 /// `Initialized` event.
 const MAX_FINALIZED_BLOCKS: usize = 16;
+/// The size of the LRU cache for pruned blocks.
+///
+/// This is the exact value of the total number of pinned blocks, and ensures
+/// that all active pruned block hashes (if any) are kept in memory.
+const LRU_CACHE_SIZE: u32 = 512;
```

Comment:
So to make sure I understand; this number must equal the number of pinned blocks we keep?

Could we instead use `chain_head::MAX_PINNED_BLOCKS` or alias this to be that, so that there's no chance these numbers can get out of sync in the future?